data/mods/ssb/scripts.ts

```diff
@@ -24,6 +24,7 @@ export const BattleScripts: ModdedBattleScriptsData = {
 
 		// Overneat gains the fairy type when mega evolving
 		if (pokemon.name === 'Overneat' && !pokemon.illusion) this.add('-start', pokemon, 'typeadd', 'Fairy');
+		if (pokemon.name === 'Frostyicelad' && !pokemon.illusion) this.add('-start', pokemon, 'typechange', 'Bug/Ice');
```

Comment:
I don't think this line is necessary. It's the same type as the base forme.

Reply:
until tooltips has it i view it as necessary personally, only reason i added it